hass_nabucasa/client.py

```diff
@@ -72,6 +72,12 @@ async def logout_cleanups(self) -> None:
     async def async_cloud_connect_update(self, connect: bool) -> None:
         """Process cloud remote message to client."""
 
+    @abstractmethod
+    async def async_cloud_connection_info(
+        self, payload: dict[Any, Any]
+    ) -> dict[Any, Any]:
```

Comment:
The typing doesn't match the typing in `async_handle_connection_info`.

Reply:
That is correct, this is consistent with all others.

Reply:
Shouldn't we correct it at least for new code?

Reply:
Better to fix it all in a separate PR?